contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -410,10 +415,35 @@ <h4>Last Updated</h4>
 
 {% include 'topnav_javascript.html' %}
 
+<!-- Modal -->
+<div class="modal fade" id="confirmDelete" tabindex="-1" role="dialog" aria-labelledby="confirmDeleteTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
```

Comment:
I think padding in the `modal-content` class may not be needed. Please check without that. Thanks.

Reply:
resolved